_tests_/App.test.jsx

```diff
@@ -1,43 +1,22 @@
 /* eslint-disable import/extensions */
 import React from 'react';
 import Enzyme from 'enzyme';
-import { shallow } from 'enzyme';
```

Comment:
I believe your tests are using shallow from Enzyme without calling out Enzyme.shallow. Is this still working for you?